COPYRIGHT

```diff
@@ -1 +1,5 @@
-Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
```

Comment:
I have no idea.  I just clicked the green button.   

Patricia Willis
www.plwillis.com


> On Jan 20, 2015, at 1:08 PM, Sulka Haro <notifications@github.com> wrote:
> 
> In COPYRIGHT:
> 
> > @@ -1 +1,5 @@
> > -Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
> Why remove copyright notice?
> 
> —
> Reply to this email directly or view it on GitHub.
>